style.css

```diff
@@ -0,0 +1,125 @@
+body {
+  margin: 0;
+  background-image: url(Images/des1.jpeg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-size: 100% 840px;
```

Comment:
- [OPTIONAL] - Your code shouldn't be repeated you have two values at the same time assigned for background-color and it's not correct, remove one of them.

Reply:
OK Teacher I will keep attention
Thanks a lot